packages/core/package.json

```diff
@@ -42,17 +42,18 @@
   },
   "dependencies": {
     "@emotion/cache": "^11.10.5",
+    "@emotion/css": "^11.10.6",
+    "@emotion/serialize": "^1.1.2",
     "@hitachivantara/uikit-react-icons": "^5.1.11",
     "@hitachivantara/uikit-react-shared": "^5.0.15",
     "@hitachivantara/uikit-styles": "^5.8.1",
+    "@mui/base": "^5.0.0-beta.3",
     "@popperjs/core": "^2.11.6",
-    "@types/react-table": "^7.7.14",
```

Comment:
Regarding the react table types, I believe we need to install them as a dependency and not a dev dependency because we need these types to pass transitively to our users since we extend them.  
I believe this is something @nantunes mentioned when we were having problems with the react table types a few weeks ago.